src/main/webapp/i18n/de/metis.json

```
@@ -92,7 +92,9 @@
                 "RANDOM": "Sonstiges",
                 "ANNOUNCEMENT": "Ankündigung",
                 "allPublicMessages": "Alle öffentlichen Nachrichten",
-                "searchResults": "Suchergebnisse für {{ search }}"
+                "searchResults": "Suchergebnisse für {{ search }}",
+                "mobileDisclaimer": "Bitte wähle erst eine Konversation aus.",
```

```suggestion
                "mobileDisclaimer": "Bitte erst eine Konversation auswählen.",
```
I think this easier to read

---

I agree, I addressed it in this [commit](https://github.com/ls1intum/Artemis/pull/9460/commits/068c060c09808233eb33695f12486d82466e22dd). 👍🏼